docs/ftldns/compile.md

```diff
@@ -48,7 +66,7 @@ git clone https://github.com/pi-hole/FTL.git && cd FTL
 If you want to build another branch and not `master`, use checkout to get to this branch, like
 
 ```bash
-git checkout development
+git checkout development-v6
 ```
 
 ## Compile the source
```

Comment:
Update sudo service pihole-FTL restart to sudo pihole restartdns

Reply:
We already checkout `development-v6`

https://github.com/pi-hole/docs/blob/3c57b3b418e0387f43a3ea0cc3b41c362a1295b7/docs/ftldns/compile.md?plain=1#L69

___


`sudo service pihole-FTL restart` should also work.

Reply:
It didn't for me at the time, but does seem to now (I can't say my setup was 100% clean). I guess it's a matter of preference whether to use the same command mentioned in the pihole --help. I'll leave that to those more knowledgeable.